sp800-63b/sec5_authenticators.md

```diff
@@ -112,7 +112,7 @@ If the out of band authenticator sends an approval message over the secondary co
 
 Out of band verifiers SHALL generate a random authentication secret with at least 20 bits of entropy using an approved random number generator. They then optionally signal the device containing the subscriber's authenticator to indicate readiness to authenticate.
 
-Due to the risk that SMS messages may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out of band verification is to be made using a SMS message on a public mobile telephone network, the verifier SHALL verify that the pre-registered telephone number being used is actually associated with a mobile network and not with a VoIP (or other software-based) service. It then sends the SMS message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using SMS is deprecated**, and may no longer be allowed in future releases of this guidance.
+Due to the risk that SMS messages may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out of band verification is to be made using a SMS message on a public mobile telephone network, the verifier SHALL verify that the pre-registered telephone number being used is actually associated with a mobile network and not with a VoIP, chat, or other software-based service. It then sends the SMS message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using SMS is deprecated**, and may no longer be allowed in future releases of this guidance.
```

Comment:
I think you make excellent points about how the verifier *can* perform validation of an SMS-based out of band factor. Perhaps MDMs could control this by restricting configuration options for forwarding SMS and installation of apps that read SMS, etc.

I didn't intend for my pull request to introduce a new requirement. My interpretation is that it's already present in the text, which currently reads "the verifier SHALL verify that the pre-registered telephone number being used is actually associated with a mobile network and not with a VoIP (_or other software-based_) service" [emphasis added]

If verifiers must check whether the number is associated with VoIP or another software service, _chat_ illustrates an example of software that's extremely common today (see examples above), albeit often through a different mechanism (relay).

With regard to your point about signal, I'm not clear that its chat service would fall under the SMS guidance. It seems to me that this would be a different type of out of band secret?